config/environments/development.rb

```diff
@@ -48,14 +48,17 @@
   # Tell Active Support which deprecation messages to disallow.
   config.active_support.disallowed_deprecation_warnings = []
 
+  # Highlight code that enqueued background job in logs.
+  config.active_job.verbose_enqueue_logs = true
+
   # Raises error for missing translations.
   # config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
   config.action_view.annotate_rendered_view_with_filenames = true
 
-  # Uncomment if you wish to allow Action Cable access from any origin.
-  # config.action_cable.disable_request_forgery_protection = true
-
+  # Raise error when a before_action's only/except options reference missing actions
```

Comment:
This comment should be the line before to the `raise_on_missing_callback_actions` I think?

Reply:
That's right!
Fixed, I've checked the `test.rb` too, which was correct.